osu.Game/Graphics/Containers/LogoFacadeContainer.cs

```diff
@@ -0,0 +1,114 @@
+        {
+            Logo = logo ?? throw new ArgumentNullException(nameof(logo));
+            this.facadeScale = facadeScale;
+            this.duration = duration;
+            this.easing = easing;
+
+            startTime = null;
+            startPosition = null;
+        }
+
+        /// <summary>
+        /// Gets the position that the logo should move to with respect to the <see cref="LogoFacade"/>.
+        /// Manually performs a conversion of the Facade's position to the Logo's parent's relative space.
+        /// </summary>
+        /// <remarks>Will only be correct if the logo's <see cref="Drawable.RelativePositionAxes"/> are set to Axes.Both</remarks>
+        /// <returns>The position that the logo should move to in its parent's relative space.</returns>
+        protected Vector2 LogoTrackingPosition()
+        {
+            Vector2 local;
+            local.X = Logo.Parent.ToLocalSpace(LogoFacade.ScreenSpaceDrawQuad.Centre).X / Logo.Parent.RelativeToAbsoluteFactor.X;
+            local.Y = Logo.Parent.ToLocalSpace(LogoFacade.ScreenSpaceDrawQuad.Centre).Y / Logo.Parent.RelativeToAbsoluteFactor.Y;
+            return local;
+        }
+
+        protected override void Update()
+        {
+            base.Update();
+
+            if (Logo == null || !Tracking)
+                return;
+
+            // Account for the scale of the actual logo container, as SizeForFlow only accounts for the sprite scale.
+            LogoFacade.Size = new Vector2(Logo.SizeForFlow * Logo.Scale.X * facadeScale);
+
+            if (LogoFacade.Parent != null && Logo.Position != LogoTrackingPosition())
+            {
+                // If this is our first update since tracking has started, initialize our starting values for interpolation
+                if (startTime == null || startPosition == null)
+                {
+                    startTime = Time.Current;
+                    startPosition = Logo.Position;
+                }
+
+                if (duration != 0)
+                {
+                    double elapsedDuration = Time.Current - startTime ?? throw new ArgumentNullException(nameof(startTime));
+
+                    var mount = (float)Interpolation.ApplyEasing(easing, Math.Min(elapsedDuration / duration, 1));
+
+                    // Interpolate the position of the logo, where mount 0 is where the logo was when it first began interpolating, and mount 1 is the target location.
+                    Logo.Position = Vector2.Lerp(startPosition ?? throw new ArgumentNullException(nameof(startPosition)), LogoTrackingPosition(), mount);
+                }
+                else
+                {
+                    Logo.Position = LogoTrackingPosition();
+                }
+            }
+        }
+
+        /// <summary>
+        /// A placeholder container that serves as a dummy object to denote another object's location and size.
+        /// </summary>
+        public class Facade : Container
```

Comment:
Should be a `Drawable`.

Should probably also throw exceptions on changing things like `RelativePositionAxes` unless they are supported.

Reply:
Not sure if the facade itself needs to have any special restrictions like this, only the logo should probably be not allowed to change its RelativePositionAxes while a tracking container is using it.

Reply:
Right. The facade should disallow size updates or whatever it's getting from the logo.